pom.xml

```diff
@@ -786,9 +786,6 @@
         <!-- Profile to compile and run tests with JDK 11 -->
         <profile>
             <id>java-11</id>
```

Comment:
Should we use <activeByDefault>false</activeByDefault> instead?

Reply:
Its already `false` by default: https://maven.apache.org/ref/3.8.1/maven-settings/settings.html